test/org/sagebionetworks/bridge/dynamodb/DynamoActivityEventDaoTest.java

```diff
@@ -53,27 +54,27 @@ public void canCrudEvent() {
         
         Map<String,DateTime> map = activityEventDao.getActivityEventMap("BBB");
         assertEquals(6, map.size());
-        assertEquals(time1, map.get("enrollment"));
-        assertEquals(time1.minusWeeks(2), map.get("two_weeks_before_enrollment"));
-        assertEquals(time1.minusMonths(2), map.get("two_months_before_enrollment"));
-        assertEquals(time2, map.get("survey:AAA-BBB-CCC:finished"));
-        assertEquals(time3, map.get("question:DDD-EEE-FFF:answered=someValue"));
-        assertEquals(time6, map.get("activity:AAA-BBB-CCC:finished"));
+        assertEquals(time1.withZone(DateTimeZone.UTC), map.get("enrollment"));
+        assertEquals(time1.withZone(DateTimeZone.UTC).minusWeeks(2), map.get("two_weeks_before_enrollment"));
+        assertEquals(time1.withZone(DateTimeZone.UTC).minusMonths(2), map.get("two_months_before_enrollment"));
+        assertEquals(time2.withZone(DateTimeZone.UTC), map.get("survey:AAA-BBB-CCC:finished"));
+        assertEquals(time3.withZone(DateTimeZone.UTC), map.get("question:DDD-EEE-FFF:answered=someValue"));
+        assertEquals(time6.withZone(DateTimeZone.UTC), map.get("activity:AAA-BBB-CCC:finished"));
```

Comment:
I changed the initial times to Moscow time, and these assertions still pass (times converted to UTC).